app-frontend/src/app/components/projects/projectItem/projectItem.html

```diff
@@ -1,14 +1,51 @@
 <div class="panel-body">
-  <div class="tag-container">
-    <h4 class="project-title">{{$ctrl.project.name}}</h4>
-    <rf-status-tag ng-if="$ctrl.status" entity-type="project" status="$ctrl.status"></rf-status-tag>
+  <rf-status-tag ng-if="$ctrl.status" entity-type="project" status="$ctrl.status"></rf-status-tag>
+  <a href ui-sref="projects.detail({projectid: $ctrl.project.id})" class="h4">
+    <h4 class="project-title" data-title="{{$ctrl.project.name}}">{{$ctrl.project.name}}</h4>
+  </a>
+  <div class="row">
+    <div class="column-12">
+
+      <!-- IF project has no scenes -->
+      <div class="project-preview-container placeholder" 
+           ng-if="!$ctrl.project.scenes && $ctrl.status !== 'CURRENT'"
```

Comment:
We don't need to check against the status here since we only care how many scenes the project has.